### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bertux**
To make it more readable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,27 +65,32 @@ WORKFLOW_INSTANCE_ID= #put id here
...
 
 ``` bash
 ./ensemble.sh instances start $WORKFLOW_INSTANCE_ID
 ```
 
 ## More Workflow Examples
 
+``` bash
 ./ensemble.sh workflows create ./samples/dca.workflow.yaml
-WORKFLOW_ID=
```
Comment (leonprou) [inline README.md:80]: this meant:
WORKFLOW_ID= #put here the received workflow id
I'd like to put this back as it was convenient for me. wdyt?

**Review by leonprou — CHANGES_REQUESTED** [review]
Hi @bertux 
Thanks for those!
Please see my comment on line 80

File: README.md (hunk 1 above)
Comment (bertux) [reply README.md:80]: Yes @leonprou  now with these details it makes more sense to keep this line but to add also the comment to explain as in other parts.

**Comment by bertux** [comment]
> Hi @bertux Thanks for those! Please see my comment on line 80

I have added back what wasn't commented and improved the comments to make it more clear.
It seems it just needs your final review and merge before it creates conflicts.

**Comment by leonprou** [comment]
> > Hi @bertux Thanks for those! Please see my comment on line 80
> 
> I have added back what wasn't commented and improved the comments to make it more clear. It seems it just needs your final review and merge before it creates conflicts.

Thanks! Looks good now!

**Pull request merged**